cmake/NeuronFileLists.cmake

```diff
@@ -28,6 +29,8 @@ set(HEADER_FILES_TO_INSTALL
     nrniv_mf.h
     nrnjava.h
     nrnoc_ml.h
+    nrnmpi.h
+    nrnmpidec.h
```

Comment:
Wondering why this was not there previously and why this is needed with binary special. Is that for `nrnmain.cpp`?

Reply:
I think it is good to be liberal with the collection of *.h files in the various folders that get copied into CMAKE_INSTALL_PREFIX/include on the off chance that some model author might need it. In these last rounds of additions, I believe that was exactly the case.

Reply:
yes needed for `nrnmain.cpp`